contrib/guix/libexec/build.sh

```diff
@@ -169,9 +209,21 @@ export PATH="${BASEPREFIX}/${HOST}/native/bin:${PATH}"
 
     # Perform basic ELF security checks on a series of executables.
     make -C src --jobs=1 check-security ${V:+V=1}
-    # Check that executables only contain allowed gcc, glibc and libstdc++
-    # version symbols for Linux distro back-compatibility.
-    make -C src --jobs=1 check-symbols  ${V:+V=1}
+
+    case "$HOST" in
+        *linux*)
+            # Check that executables only contain allowed gcc, glibc and libstdc++
+            # version symbols for Linux distro back-compatibility.
+            make -C src --jobs=1 check-symbols  ${V:+V=1}
+            ;;
```

Comment:
I added dylib checking for PE binaries in #18395. Can we add the relevant case and `make -C src check-symbols` call here too?

Reply:
Fixed in e63cb74749aa3b82c04fc9dacd963d3de00784b9